Tests/ScipioKitTests/CacheSystemTests.swift

```diff
@@ -83,17 +83,17 @@ final class CacheSystemTests: XCTestCase {
     }
 
     func testCacheKeyForRemoteAndLocalPackageDifference() async throws {
-        let fileSystem = localFileSystem
+        let fileSystem = LocalFileSystem.default
```

Comment:
optional. It's more prefer

```suggestion
        let fileSystem: LocalFileSystem = .default
```